cypress/e2e/api_login_tests.cy.js

```diff
@@ -0,0 +1,18 @@
+import { LoginPage } from "../page-objects/login_page";
+
+describe("Login API test Tegb", () => {
+  beforeEach(() => {
+    new LoginPage()
+      .openTegb()
+      .typeUsername(Cypress.env("tegb_username"))
+      .typePassword(Cypress.env("tegb_password"))
+      .clickLogin();
+  });
+
+  it("access_token is present in Login API and status is 201", () => {
+    cy.wait("@login_api").then((loginApi) => {
```

Comment:
Provoláváš tady kontrolu v rámci frontend loginu. Test měl být o tom, že to API ručně provoláš.